### DISCUSSION TITLE
Docs new top div cause can't see left, right sidebar content

### DISCUSSION THREAD

**Discussion by tommy66374** [livewire/livewire#7350] [4. General]
The `New screencasts are 30% off for Black Friday...` div cause the left, right sidebar content move down

The easy way just add a bottom padding in both sidebar div ?

**Comment by joshhanley** [comment]
@tommy66374 looks fine on my device, can you provide a screenshot of what you mean? Thanks!

**Comment by tommy66374** [comment]
> @tommy66374 looks fine on my device, can you provide a screenshot of what you mean? Thanks!

In Firefox, i scroll down the both left, right sidebar to the bottom

![Firefox-Livewire-Docs](https://github.com/livewire/livewire/assets/80481558/09f4d5c3-2554-4f28-9c4e-40b77e9ad683)

**Reply by joshhanley** [reply]
@tommy66374 ah yep, thanks for reporting, will get someone to have a look 🙂
